Configuration/AlCa/python/autoCond.py

```diff
@@ -20,9 +20,11 @@
     # GlobalTag for MC production (Heavy Ions collisions) with optimistic alignment and calibrations for Run2
     'run2_mc_hi'        :   '80X_mcRun2_HeavyIon_v3',
     # GlobalTag for Run1 data reprocessing
-    'run1_data'         :   '80X_dataRun2_v4',
+    'run1_data'         :   '80X_dataRun2_v5',
```

Comment:
git shows the 1st diff indeed due to the commit you are referring to.. The other commit that i reverted was the second part, which fixed the conflict (which is why this is no longer merging)